warehouse/ingest-core/src/test/java/datawave/ingest/data/config/XMLFieldConfigHelperTest.java

```diff
@@ -230,6 +237,42 @@ public void testSameLengthOverlappingRegexPrecise() throws Exception {
         assertFalse(helper.isIndexedField("BANANA"));
     }
 
+    @Test
+    public void testCombineFile() throws Exception {
```

Comment:
TODO:  add a test case for one-to-many combined mappings, i.e.  name="A" combine="B,C,D"  should generate an A for each element.